cluster/gce/gci/configure-helper.sh

```diff
@@ -2436,6 +2453,9 @@ EOF
   if [[ "${NETWORK_POLICY_PROVIDER:-}" == "calico" ]]; then
     setup-addon-manifests "addons" "calico-policy-controller"
 
+    setup-addon-custom-yaml "addons" "calico-policy-controller" "calico-node-daemonset.yaml" "${CUSTOM_CALICO_NODE_DAEMONSET_YAML}"
```

Comment:
where is CUSTOM_CALICO_NODE_DAEMONSET_YAML defined?

Reply:
Changed to refer it as ${CUSTOM_CALICO_NODE_DAEMONSET_YAML:-} to allow it is unbound if not specified.